package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-beacon",
-  "version": "0.8.3",
+  "version": "0.8.6",
```

Comment:
I haven't been a bit loose about updating the version numbers, only doing it when I publish, which is why they aren't always consecutive in the pull requests.